fastlane/metadata/android/ru-RU/video.txt

```diff
@@ -1 +1,9 @@
 https://youtu.be/yF60i6nt4Cw
+
+Many different levels - ��������� ������������� �������
```

Comment:
The video.txt should just have one line containing the link to YouTube. Please check why you have other text and that symbol � here too.